plugins/render/api_key.go

```diff
@@ -0,0 +1,76 @@
+package render
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+type Config struct {
+	Version           int
+	SSHPreserveHosts  bool `yaml:"sshPreserveHosts"`
+	Profiles          map[string]Profile
+}
+
+type Profile struct {
+	DefaultRegion string `yaml:"defaultRegion"`
+	APIKey        string `yaml:"apiKey"`
+}
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://render.com/docs/api_key"),
+		ManagementURL: sdk.URL("https://console.render.com/user/security/tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.APIKey,
+				MarkdownDescription: "API Key used to authenticate to Render.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 30,
+					Charset: schema.Charset{
+						Uppercase: true,
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer: importer.TryAll(
+			importer.TryEnvVarPair(defaultEnvVarMapping),
+			TryRenderConfigFile(),
+		),
+	}
+}
+
+var defaultEnvVarMapping = map[string]sdk.FieldName{
+	"RENDER_API_KEY": fieldname.APIKey,
```

Comment:
I couldn't find this envvar documented anywhere for the CLI, only for the Terraform provider. Is there some documentation that I could read up on?

Reply:
I have made the dictionary empty.